decidim-core/lib/decidim/api/interfaces/authorable_interface.rb

```diff
@@ -12,11 +12,12 @@ module AuthorableInterface
       end
 
       def author
-        if object.respond_to?(:normalized_author)
-          object&.normalized_author
-        elsif object.respond_to?(:creator_identity)
-          object&.creator_identity
-        end
+        author = if object.respond_to?(:normalized_author)
+                   object&.normalized_author
+                 elsif object.respond_to?(:creator_identity)
+                   object&.creator_identity
+                 end
+        author if author.is_a?(Decidim::User) || author.is_a?(Decidim::UserGroup)
```

Comment:
It may be a preference but I believe we could also do this:
```suggestion
        author if author.is_a?(Decidim::UserBaseEntity)
```